vsts/scripts/tagBuildImagesForRelease.sh

```diff
@@ -11,16 +11,11 @@ declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && cd .. && pwd )
 source $REPO_DIR/build/__variables.sh
 
 outPmeFileMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/oryxprodmcr-build-images-mcr.txt"
-outNonPmeFileMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/oryxmcr-build-images-mcr.txt"
```

Comment:
There still have lots of lines involving `NonPme` were not removed from these four scripts. I suggest you search the keyword in these files and make sure the PR's removed all lines.

Reply:
thanks I will remove this and anything that are remaining as well